lib_standard_app/Makefile.standard_app

```diff
@@ -0,0 +1,93 @@
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#*******************************************************************************
+
+STANDARD_APP_FLAGS = 0x000
+
+#####################################################################
+#                            BLUETOOTH                              #
+#####################################################################
+ifeq ($(ENABLE_BLUETOOTH), 1)
+    STANDARD_APP_FLAGS = 0x200  # APPLICATION_FLAG_BOLOS_SETTINGS
+    STANDARD_APP_DEFINES += HAVE_BLE BLE_COMMAND_TIMEOUT_MS=2000 HAVE_BLE_APDU
+    STANDARD_SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
+endif
+
+#####################################################################
+#                               DEBUG                               #
+#####################################################################
+ifneq ($(DEBUG), 0)
+    STANDARD_APP_DEFINES += HAVE_PRINTF
+    ifeq ($(TARGET_NAME),TARGET_NANOS)
+        STANDARD_APP_DEFINES += PRINTF=screen_printf
+    else
+        STANDARD_APP_DEFINES += PRINTF=mcu_usb_printf
+    endif
+else
+        STANDARD_APP_DEFINES += PRINTF\(...\)=
+endif
+
+#####################################################################
+#                        IO SEPROXY BUFFER SIZE                     #
+#####################################################################
+ifneq ($(DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE), 1)
+    ifeq ($(TARGET_NAME), TARGET_NANOS)
+        STANDARD_APP_DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=128
+    else
+        STANDARD_APP_DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=300
+    endif
+endif
+
+#####################################################################
+#                          STANDARD DEFINES                         #
+#####################################################################
+STANDARD_APP_DEFINES += $(DEFINES_LIB)
+STANDARD_APP_DEFINES += APPNAME=\"$(APPNAME)\"
+STANDARD_APP_DEFINES += APPVERSION=\"$(APPVERSION)\"
+STANDARD_APP_DEFINES += MAJOR_VERSION=$(APPVERSION_M) MINOR_VERSION=$(APPVERSION_N) PATCH_VERSION=$(APPVERSION_P)
+STANDARD_APP_DEFINES += OS_IO_SEPROXYHAL
+
+ifneq ($(DISABLE_STANDARD_APP_DEFINES), 1)
+    STANDARD_APP_DEFINES += HAVE_SPRINTF HAVE_SNPRINTF_FORMAT_U
+    STANDARD_APP_DEFINES += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=6 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
+    STANDARD_APP_DEFINES += USB_SEGMENT_SIZE=64
+    STANDARD_APP_DEFINES += BLE_SEGMENT_SIZE=32
+    STANDARD_APP_DEFINES += HAVE_WEBUSB WEBUSB_URL_SIZE_B=0 WEBUSB_URL=""
+
+ifneq ($(TARGET_NAME), TARGET_STAX)
+    STANDARD_APP_DEFINES += HAVE_UX_FLOW
+endif
+
+endif
```

Comment:
I'm wondering what you think to cut it in:
- DISABLE_SNPRINTF
- DISABLE_USB
- DISABLE_WEBUSB
- DISABLE_BAGL_UX_FLOW